### ISSUE TITLE
API Properties for exteded classes are ignored

### ISSUE THREAD

**Issue by Karankang007** [ui-jar/ui-jar#15]
If a component extends another component, input bindings and other properties for base component are ignored.

**Comment by nording** [comment]
Thanks for the report! I will fix this, it will be fixed in the next release. WIP

**Comment by nording** [comment]
RC-1 is now released!

**Comment by Karankang007** [comment]
I can still reproduce this in v1.0.0

**Comment by nording** [comment]
@Karankang007 Could you provide an example component to reproduce this?

**Comment by Karankang007** [comment]
Hi @nording, do you still need an example to repro this?

**Comment by Karankang007** [comment]
**Base Component:**
```
import { Component, Input } from '@angular/core';

@Component({
    selector: 'base-test',
    template: `<span>{{value}}</span>`
})
export class BaseTestComponent {
    @Input()
    public value: string;
}
```

**Derived Component**
```
import { Component, OnInit } from '@angular/core';
import { BaseTestComponent } from './baseTest.component';

@Component({
    selector: 'derived-test',
    template: `<span [style.color]="color">{{value}}</span>`
})
export class NameComponent extends BaseTestComponent {
    @Input()
    public color: string;
}
```

**Comment by nording** [comment]
Thanks for repro example!

**Comment by nording** [comment]
I'm trying your example, when I'm setting "value" and "color" in the test I see both color and value properties in in the example. It's also available in "view-source", in API-tab I see "@Input() color" and "@Input() value".

@Karankang007 Could you provide the test you use to repro this?

The test code I used is this:
```
import { TestBed, async } from '@angular/core/testing';
import { NameComponent } from './derived.component';

describe('NameComponent', () => {
    beforeEach(async(() => {
        /** 
         * @uijar NameComponent
         **/
        TestBed.configureTestingModule({
            declarations: [
                NameComponent
            ],
        }).compileComponents();
    }));

    /** @uijarexample Repro example */
    it('lorum ipsum dolor sit...', async(() => {
        const fixture = TestBed.createComponent(NameComponent);
        fixture.componentInstance.value = 'my value';
        fixture.componentInstance.color = 'green';
        ...
    }));
});
```

**Comment by Karankang007** [comment]
@nording 

I have created a repository for you to see this issue: https://github.com/Karankang007/uijar-example-1

The progress bar component uses  a base class with color input.

The API tab for UI-Jar is not show the "color" input property.

**Comment by nording** [comment]
Fix is available in v1.1.2 - now available at npm!

**Comment by nording** [comment]
Fix for base classes with generics is available in v1.1.4.
